model/lifecycle.go

```diff
@@ -538,7 +538,8 @@ func addTasksToBuild(ctx context.Context, creationInfo TaskCreationInfo) (*build
 		if !bvtu.HasSpecificActivation() {
 			continue
 		}
-		activateTaskAt, err := creationInfo.ProjectRef.GetActivationTimeForTask(bvtu)
+		// Use the version create time to ensure we're being consistent
+		activateTaskAt, err := creationInfo.ProjectRef.GetActivationTimeForTask(bvtu, creationInfo.Version.CreateTime)
```

Comment:
Do we need to test that this sets the activation time to the version creation time for tasks?

Reply:
Added a test for createVersionItems that verifies using the version create time